tensorboard/plugins/graph/tf_graph_dashboard/tf-graph-dashboard.html

```diff
@@ -145,9 +142,33 @@ <h3>No graph definition files were found.</h3>
  */
 const RUN_STORAGE_KEY = 'run';
 
+/**
+ * TODO(stephanwlee): Convert this to proper type when converting to TypeScript.
+ * @typedef {{
+ *   tag: ?string,
+ *   displayName: string,
+ *   conceptualGraph: boolean,
+ *   opGraph: boolean,
+ *   profile: boolean,
+ * }}
+ */
+ const TagItem = {};
+
+/**
+ * TODO(stephanwlee): Convert this to proper type when converting to TypeScript.
+ * @typedef {{
+ *   name: string,
+ *   tags: Arary<!TagItem>,
```

Comment:
s/Arary/Array/: is this not producing a compiler warning/error?

also, `tags` is not nullable, right? `!Array<!TagItem>`